test/Microsoft.ML.Benchmarks.Tests/Microsoft.ML.Benchmarks.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.ML.Benchmarks\Microsoft.ML.Benchmarks.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.PerformanceTests\Microsoft.ML.PerformanceTests.csproj" />
 
```

Comment:
Should the csproj files be renamed to reflect the change from Benchmark -> PerformanceTests?

Reply:
Yes definitely. I could not decide on a name other than `Microsoft.ML.PerformanceTests.Tests`, so I didn't change it for now.